proto/viam/app/v1/app.proto

```diff
@@ -161,7 +161,6 @@ message RobotPartHistoryEntry {
   RobotPart old = 4 [(tagger.v1.tags) = "bson:\"old\" json:\"old\""];
 }
 
-// Organizations
```

Comment:
undo?

Reply:
I need a file in proto/** change for it to kick off the workflow. And this wasn't a great comment to begin with.